src/components/header/HeaderComponent.tsx

```diff
@@ -88,6 +90,7 @@ export default class HeaderComponent extends React.Component<Props, State> {
     const {
       title,
       subTitle,
+      subTitleSecondLine,
```

Comment:
Title should fit in one line.
To remove.

Reply:
It can not fit on one line if we keep the license plate, some car names are already too long (Tesla Model 3 Long Range Dual Motor).

Reply:
@LucasBrazi06 what do you think ?

Reply:
@LucasBrazi06 what do you think ?

Reply:
Let the car name with truncated ... if too long